controllers/pod_controller.go

```diff
@@ -94,6 +104,71 @@ func (r *PodReconciler) makeSpiffeId(obj ObjectWithMetadata) string {
 	return r.makeSpiffeIdForPod(obj.(*corev1.Pod))
 }
 
+func (r *PodReconciler) mungeIp(ip string) string {
+	if strings.Contains(ip, ".") {
+		return strings.Replace(ip, ".", "-", -1)
```

Comment:
No need to check ':' below in this case?

Reply:
ip is either going to be IPv4 (contains .) or IPv6 (contains : )